### PR TITLE
[DC-1475]  Add reCaptcha validation

### PR DESCRIPTION
**Pull request by mina-sealink**
### Why
WA are getting 1000's of spam emails from their contact form. We need to add reCaptcha to our contact forms. This needs to be off by default with the option to turn it on per form.

### Test
1. Change the  'Formerly' plugin's version on the composer.json (One SeaLink)
`"sealink/formerly": "1.7.0-alpha.0",`
2. Run `composer clear-cache` and `composer update formerly`
3. Run One SeaLink project and go to CMS 
4. Click the 'Formerly' on the left menu bar and select an form
5. Check the 'Use Google reCaptcha'
6. Input the Google keys for reCaptcha
You can check the keys here: https://www.google.com/recaptcha/admin
dsm.sealink@gmail.com account.

### Result
**CMS**
screen shot 2018-08-02 at 11 51 25 am

screen shot 2018-08-02 at 11 51 56 am

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Formerly
+
+## Unreleased
+
+* [DC-1475] Modify Formerly plugin to add reCaptcha validation
```
Comment (tlclink) [inline CHANGELOG.md:5]: Why is this entire file new?

Comment (mina-sealink) [reply CHANGELOG.md:5]: I created it because we did not have the file.

Comment (tlclink) [reply CHANGELOG.md:5]: strange thats unexpected

Comment (mina-sealink) [reply CHANGELOG.md:5]: We used the 'readme.md' file to display update lists

**Comment by tlclink** [comment]
Just a note for other reviewers that there is a number of tab -> space transformations in this PR so its best to review commit by commit
@mina-sealink for future reference if you do a code spacing cleanup its best to do it in the first commit instead of the last commit since otherwise your likely to end up with a few spacing cleanups with your real code changes making it slightly more work to review

**Review by tlclink — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -55,6 +55,8 @@ regular `ElementCriteriaModel`s. For example, to list submissions to the
 ```
 
 ## Updates
+* 1.7.0
```
Comment (dwaring) [inline README.md]: Do we want to remove this section of the readme?
Otherwise we're needing to update both the (newly created) changelog, plus here.

Comment (mina-sealink) [reply README.md]: Removed them!

**Review by dwaring — APPROVED** [review]

**Pull request merged**